pkg/go.mod

```diff
@@ -28,15 +28,10 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
 
-replace (
-	go.etcd.io/etcd/api/v3 => ../api
-	go.etcd.io/etcd/client/pkg/v3 => ../client/pkg
-)
-
-// Bad imports are sometimes causing attempts to pull that code.
-// This makes the error more explicit.
 replace (
 	go.etcd.io/etcd => ./FORBIDDEN_DEPENDENCY
+	go.etcd.io/etcd/api/v3 => ./FORBIDDEN_DEPENDENCY
+	go.etcd.io/etcd/client/pkg/v3 => ../client/pkg
```

Comment:
Can you move the `go.etcd.io/etcd/client/pkg/v3 => ../client/pkg` into a separate replace block above, and keep the comment "Bad imports are sometimes cause attempts to ..." for this replace block?

Reply:
Good Idea.